src/undo/DeleteUndoAction.h

```diff
@@ -35,6 +32,7 @@ class DeleteUndoAction: public UndoAction {
     string getText() override;
 
 private:
-    GList* elements = nullptr;
+    std::multiset<PageLayerPosEntry<Element>, decltype(&PageLayerPosEntry<Element>::cmp)> elements{
+            PageLayerPosEntry<Element>::cmp};
```

Comment:
Like below this violates the Compare Requirement. (Reference below)
```suggestion
    std::multiset<PageLayerPosEntry<Element>> elements{};
```